examples/patterns/tables/table-expanding.html

```diff
@@ -0,0 +1,169 @@
+                    </div>
+                    <div class="col-6">
+                        <img src="http://placehold.it/1024x325" alt="">
+                    </div>
+                </div>
+            </td>
+        </tr>
+        <tr role="row">
+            <td role="rowheader" aria-label="Name">Melon</td>
+            <td role="gridcell" aria-label="Users" class="u-align--right">12</td>
+            <td role="gridcell" aria-label="Units" class="u-align--right">23</td>
+            <td role="gridcell" class="u-align--right">
+                <button class="u-toggle" aria-controls="#expanded-row-2" aria-expanded="false">Show</button>
+            </td>
+            <td id="expanded-row-2" class="p-table-expanding__panel" aria-hidden="true">
+                <div class="row">
+                    <div class="col-6">
+                        <h3>Expanding table cell</h3>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Consequuntur cum dicta beatae nostrum eligendi similique earum, dolorem fuga quis, sequi voluptates architecto ipsa dolorum eaque rem expedita inventore voluptas odit aspernatur alias molestias facere, eum accusamus dolor, assumenda. Eaque, id! Dolorem perferendis reprehenderit eum, odio minima ad commodi earum non, iste suscipit.</p>
+                    </div>
+                    <div class="col-6">
+                        <img src="http://placehold.it/1024x325" alt="">
+                    </div>
+                </div>
+            </td>
+        </tr>
+        <tr role="row">
+            <td role="rowheader" aria-label="Name">Apple</td>
+            <td role="gridcell" aria-label="Users" class="u-align--right">9</td>
+            <td role="gridcell" aria-label="Units" class="u-align--right">17</td>
+            <td role="gridcell" class="u-align--right">
+                <button class="u-toggle" aria-controls="#expanded-row-3" aria-expanded="false">Show</button>
+            </td>
+            <td id="expanded-row-3" class="p-table-expanding__panel" aria-hidden="true">
+                <div class="row">
+                    <div class="col-6">
+                        <h3>Expanding table cell</h3>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Consequuntur cum dicta beatae nostrum eligendi similique earum, dolorem fuga quis, sequi voluptates architecto ipsa dolorum eaque rem expedita inventore voluptas odit aspernatur alias molestias facere, eum accusamus dolor, assumenda. Eaque, id! Dolorem perferendis reprehenderit eum, odio minima ad commodi earum non, iste suscipit.</p>
+                    </div>
+                    <div class="col-6">
+                        <img src="http://placehold.it/1024x325" alt="">
+                    </div>
+                </div>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+<h2>Example JavaScript</h2>
```

Comment:
You can remove this JavaScript in the markup as example is will pick up the doc script below.